templates/terraform/examples/base_configs/test_file.go.erb

```diff
@@ -27,13 +27,34 @@ import (
 func TestAcc<%= test_slug -%>(t *testing.T) {
   t.Parallel()
 
+	context := map[string]interface{} {
+	<% example.test_env_vars.each do |var_name, var_type| -%>
```

Comment:
This block seems like a candidate for a subtemplate? It's a large chunk of lines that are distracting to first-time file readers.

Reply:
Fair enough! My feeling is that this block seems disproportionately large at first read but I can understand the argument the other way as well.